From 5f83c373c89a0e91a75a53184ce896426385293c Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 28 Apr 2014 14:30:51 +0200
Subject: [LIBREPORT PATCH 05/10] less confusing label for 'upload data' in
 Anaconda

Because 'Upload the problem data' might evoke the automatic bug creation
in the Bugzilla.

We know that users (an me as well) read fast and do not read the entire
sentences. Hence the first words of a button's label must express the
purpose of that button.

Related to #259

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/workflows/workflow_AnacondaUpload.xml.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/workflows/workflow_AnacondaUpload.xml.in b/src/workflows/workflow_AnacondaUpload.xml.in
index a98a536..541c898 100644
--- a/src/workflows/workflow_AnacondaUpload.xml.in
+++ b/src/workflows/workflow_AnacondaUpload.xml.in
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workflow>
-    <_name>Upload the problem data to a server</_name>
-    <_description>Analyze the problem locally and upload the data via scp or ftp</_description>
+    <_name>Export the problem data for manual reporting</_name>
+    <_description>Upload the data via scp or ftp to a remote destination</_description>
 
     <events>
         <event>report_Uploader</event>
     </events>
-</workflow>
\ No newline at end of file
+</workflow>
-- 
1.8.3.1

